templates/layout/examples/dashboard.php

```diff
@@ -16,7 +16,7 @@
 <body <?= $this->fetch('tb_body_attrs') ?>>
     <header class="navbar navbar-dark sticky-top bg-dark flex-md-nowrap p-0 shadow">
         <?= $this->Html->link(
-            Configure::read('App.title'),
+            Configure::read('App.title', ''),
```

Comment:
```suggestion
            Configure::read('App.title', 'Bootstrap UI'),
```
Without some text the link would be invisible/un-clickable.